src/components/common/TmSessionImport.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <SessionFrame>
     <div class="session">
+      <a v-if="inExtension" @click="goBack">
```

Comment:
this is a good implementation! 

instead of duplicating the back button in both session screens, should we make this 1 component that can be tested on its own?

Reply:
Thanks :)
Do you think just adding it to the SessionFrame with conditionals to display based on the route would make sense? A separate component may be the answer but the back buttons are only in two places. @jbibla

Reply:
Do you guys think that it makes sense to have the back button only in the extension? If there is a need for it in the extension, wouldn't it make for a better UX in the browser as well?

Reply:
This works well